src/components/modules/RecentAndTrending/index.tsx

```diff
@@ -0,0 +1,158 @@
+                                    {...this.MainPostImageProps(isDesktop)}
+                                    {...mainPost}
+                                    />
+                                </OverlapScaffold.Main>
+                
+                                <OverlapScaffold.Overlap>
+                                    <ModuleBox offsetDirection={OFFSET_DIRECTION.RIGHT} patternColor={sectionColor}>
+                                    {isDesktop ? (
+                                        <PostGrid order={1} posts={secondaryPosts} />
+                                    ) : (
+                                        <PostGrid order={2} itemsPerRow={1} posts={secondaryPosts} />
+                                    )}
+                                    </ModuleBox>
+                                </OverlapScaffold.Overlap>
+                            </OverlapScaffold>
+                        </Row>
+                      </View>
+
+                      <View style={[isDesktop ? styles.trendingDesktopContainer : styles.trendingMobileContainer]}>
+                          {isDesktop ? (
+                                <Text style={[styles.headerText, localFontStyles, !isDesktop ? styles.trendingMobileTextStyles : {}]}>Trending in {capitalize(tag)}</Text>
+                            ) : (
+                                <View style={{paddingHorizontal: '40px'}}>
+                                    <Pattern color={COLOR_MAP.GREEN}>
+                                        <View style={[styles.colorBackground, { padding: '15px', backgroundColor: COLOR_MAP.GREEN, flexDirection: 'row', justifyContent: 'center' }]}>
+                                            <Text style={styles.text}>Trending in {capitalize(tag)}</Text>
+                                            <View style={{width: '20%'}}>
+                                                <Image width={45} height={27} source={{ uri: TrendingLine }} />
+                                            </View>
+                                        </View>
+                                    </Pattern>
+                                </View>
+                          )}
+                          {isDesktop ? (
+                                <PostGrid order={3} itemsPerRow={1} posts={postsTwo} />
+                            ) : (
+                                <View style={{paddingHorizontal: '15px', marginTop: 30}}>
+                                    <ModuleBox>
+                                        <PostGrid order={4} itemsPerRow={1} posts={postsTwo} />
+                                    </ModuleBox>
+                                </View>
+                            )}
+                      </View>
+                    </Container>
+                  );
+                }
+            }
+            </Responsive>
+        );
+    }
+}
+
+// Capitalize first letter of tag name
+const capitalize = (str: string) => {
```

Comment:
might as well move this function to the utils module

Reply:
Moved to utils